test/Test/Time/Property.hs

```diff
@@ -41,10 +51,21 @@ unitChooser 8 t = MkAnyTime $ Time @WeekUnit        t
 unitChooser 9 t = MkAnyTime $ Time @FortnightUnit   t
 unitChooser _ _ = error "Impossible happened"
 
--- | Verifier for 'AnyTime'.
+-- | Verifier for 'AnyTime' @read . show = id@.
 verifyAnyTime :: (MonadTest m) => AnyTime -> m ()
 verifyAnyTime (MkAnyTime t) = read (show t) === t
 
+-- | Verifier for 'toUnit'.
+verifyToUnit :: forall m . (MonadTest m) => AnyTime -> AnyTime -> m ()
+verifyToUnit (MkAnyTime t1) (MkAnyTime t2) = asd t1 t2
+  where
+    asd :: forall (a :: Rat) (b :: Rat) .
```

Comment:
Maybe better to rename type variables to `unitTo` and `unitFrom` to make this function clearer.